<ui version="4.0" >
 <class>QmmpFileDialog</class>
 <widget class="QDialog" name="QmmpFileDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add Files</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Look in:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lookInComboBox" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="upToolButton" >
       <property name="text" >
        <string>Up</string>
       </property>
       <property name="icon" >
        <iconset resource="images/images.qrc" >:/cdup.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="listToolButton" >
       <property name="toolTip" >
        <string>List view</string>
       </property>
       <property name="text" >
        <string>lst</string>
       </property>
       <property name="icon" >
        <iconset resource="images/images.qrc" >:/list.png</iconset>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="iconToolButton" >
       <property name="toolTip" >
        <string>Icon view</string>
       </property>
       <property name="text" >
        <string>icn</string>
       </property>
       <property name="icon" >
        <iconset resource="images/images.qrc" >:/detail.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="fileListView" >
     <property name="editTriggers" >
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragEnabled" >
      <bool>true</bool>
     </property>
     <property name="dragDropMode" >
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="alternatingRowColors" >
      <bool>false</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="movement" >
      <enum>QListView::Static</enum>
     </property>
     <property name="layoutMode" >
      <enum>QListView::Batched</enum>
     </property>
     <property name="viewMode" >
      <enum>QListView::ListMode</enum>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>File name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fileNameLineEdit" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addPushButton" >
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePushButton" >
       <property name="text" >
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images/images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>closePushButton</sender>
   <signal>clicked()</signal>
   <receiver>QmmpFileDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>459</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel" >
     <x>153</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
